homeassistant/components/proxmoxve/__init__.py

```diff
@@ -111,91 +115,100 @@ def build_client() -> ProxmoxAPI:
                 continue
             except SSLError:
                 _LOGGER.error(
-                    'Unable to verify proxmox server SSL. Try using "verify_ssl: false"'
+                    'Unable to verify proxmox server SSL. "\
```

Comment:
I'm not sure the slash is needed

Reply:
It is needed for a string spanning multiple lines